app/mailers/course_vacancies_updated_email_mailer.rb

```diff
@@ -17,6 +17,22 @@ def course_vacancies_updated_email(course:, user:, datetime:, vacancies_filled:)
     mail(to: user.email)
   end
 
+  def course_vacancies_partially_updated_email(course:, user:, datetime:, vacancies_opened:, vacancies_closed:)
```

Comment:
Is there a better name for `course_vacancies_partially_updated_email` 🤔

Reply:
A few options:

1) Just shorten and remove the updated/email word. We already have this in the class name `CourseVacanciesUpdatedEmailMailer`, so you can build on this with a method name like `partially_updated`.
2) Building on from the first point, use namespacing for more descriptive/simple email classes which deal with one particular type of email ie `NotificationEmails::CourseVacancies::PartiallyUpdated`

Just some options :)

Reply:
Like those suggestions @darokel - will make the changes